### DISCUSSION TITLE
Are exported documents imported back at the right path if it was in a nested collection?

### DISCUSSION THREAD

**Discussion by nonoumasy** [benyap/firestore-backfire#348] [Q&A] [answered]
I have a collection called 'stories' that have several 'story' documents and each one has an 'events' collection. When exported using this library, it will export a flat list of documents with these paths:

stories/zfwnBvXn452SGunMvjKu
stories/zfwnBvXn452SGunMvjKu/characters/6Bybkn2pgZHxjP1FqXvU
stories/zfwnBvXn452SGunMvjKu/characters/8ylBdaBIZBRn48EIkfnU
stories/zfwnBvXn452SGunMvjKu/events/11VY8Jm9nnQjrxJtwmuv
stories/zfwnBvXn452SGunMvjKu/events/3aoRyVt3MbAEdtskBvSV
stories/zfwnBvXn452SGunMvjKu/events/3dGHJjMUCjaLGbLSCiC1

When these documents are imported back into Firestore, will they be imported back with the same structure as before?

**Comment by benyap** [comment, marked as answer]
Yes. Your documents will be imported back using those paths, which will place them at the same place they were exported from.
